packages/cdktf-cli/lib/get/generator/models/attribute-type-model.ts

```diff
@@ -30,6 +30,7 @@ export class AttributeTypeModel {
   public isMap: boolean;
   public struct?: Struct;
   public level?: number;
+  // FIXME: do we need an isNested here? maybe for creating different outcomes for the TF JSON?
```

Comment:
Other than the actual proto definition file, I'm having trouble finding documentation on the changes to know if anything is different.

Reply:
I have not found anything yet. There is an internal RFC describing the protocol v6 (which seems to be only adding the nested type) but that is a bit broader.